sharding-core/sharding-core-preprocessor/src/main/java/org/apache/shardingsphere/core/optimize/segment/insert/InsertValueContext.java

```diff
@@ -47,13 +47,13 @@
     
     private final List<Object> parameters;
     
-    public InsertValueContext(final Collection<ExpressionSegment> assignments, final List<Object> parameters, final int parametersOffset) {
+    public InsertValueContext(final Collection<? extends ExpressionSegment> assignments, final List<? extends Object> parameters, final int parametersOffset) {
```

Comment:
should not use List<? extends Object>, just keep original signature to keep consist with user parameters.